.github/files/lint-project-structure.sh

```diff
@@ -165,7 +165,7 @@ for PROJECT in projects/*/*; do
 			echo "::error file=$PROJECT/package.json${LINE:-$LINE2}::Set \`.repository.type\` to \"git\", as the monorepo is a git repository."
 		fi
 		URL="$(jq -r '.url' <<<"$JSON")"
-		if [[ "$URL" != "https://github.com/Automattic/jetpack.git" && "$URL" != "https://github.com/Automattic/jetpack" ]]; then
+		if [[ "$URL" != "git+https://github.com/Automattic/jetpack.git" && "$URL" != "https://github.com/Automattic/jetpack" ]]; then
```

Comment:
Possibly the second one should also be changed?

Reply:
Or maybe remove it entirely, if we really want to match what `npm pkg fix` does.

Reply:
I was assuming `.git` suffix would mean git-consumable, whereas without it would be a normal URL. But now that you quoted that line and showed that `git clone` doesn't work I'm even more confused.

I was trying to track down if/when this changed, but couldn't. I found the example in the documentation changed here:
https://github.com/npm/cli/pull/7615

I did find there's a `noGitPlus` option on `hosted-git-info`, which seems to be what provides the normalized URL info:
https://github.com/npm/hosted-git-info/

Reply:
It seems to be something internal where `git+https` or `git+ssh` but, in either case, it knows it is a github link, so it doesn't look like it actually matters?

I could see where an enterprise version of GH or other self-hosted git might have more of a need for it.

I'm fine dropping this as not needed since npm is figuring it out just fine it seems like. It doesn't really seem to be adding value unless I'm missing something.